app/controllers/places_controller.rb

```diff
@@ -53,9 +53,12 @@ def show
 
 protected
 
```

Comment:
These lines appear to form a hidden api contract with frontend: https://github.com/alphagov/frontend/blob/a4c83c956e1b2ac152bd7701941b2b4affcb8f45/app/models/imminence_response.rb

Leaving this as it is at the moment should handle frontend's required behaviour, but I will test this on integration. We should add a tech debt card to work out between us and Navigation a better way to do this (make the status more meaningful, for example?)